src/Core/Geographic/Coordinates.js

```diff
@@ -224,8 +224,32 @@ export function convertValueToUnit(unitIn, unitOut, value) {
 function Coordinates(crs, ...coordinates) {
     this._values = new Float64Array(3);
     this.set(crs, ...coordinates);
+
+    Object.defineProperty(this, 'surfaceNormal',
+        {
+            configurable: true,
+            get: () => {
+                const result = this._normal || this.computeSurfaceNormal();
+                return result;
+            },
+        });
 }
 
+Coordinates.prototype.computeSurfaceNormal = function computeSurfaceNormal() {
```

Comment:
Move this function out of the prototype: the user don't need to call it explicitely so let's not expose it.